src/caselawclient/models/documents.py

```diff
@@ -216,6 +216,14 @@ def get_latest_manifestation_datetime(
         else:
             return max(events)
 
+    def get_latest_manifestation_type(self) -> str:
+        if self.get_latest_manifestation_datetime(
+            "transform"
+        ) > self.get_latest_manifestation_datetime("tna-enriched"):
```

Comment:
You might need to check if it's valid to compare None to a date time -- you might be better using the string version.

Reply:
gave it a little refactor which hopefully makes things clearer!